spec/services/facility_stats_service_spec.rb

```diff
@@ -5,23 +5,6 @@
   let(:december) { Date.parse("12-01-2020").beginning_of_month }
   let(:period) { Period.month(december) }
 
-  def setup_for_size(size, month = december)
```

Comment:
I'm removing this for a few reasons. 
1) As I get more accustomed to our communal style, I see that we mostly set up our tests within individual tests rather than setting up complex structures for testing. 
2) This was too rigid and didn't allow enough variation.
3) This doesn't work with the tests I needed to set up for testing the registrations count, so this was only being used in one test.